server/app.ts

```diff
@@ -5,7 +5,8 @@ import cookieParser from 'cookie-parser';
 import { join } from 'path';
 import morgan from 'morgan';
 import {
-  Announcement, Advertisement, BillModel, FlatModel, ServiceModel,
+  ComplaintsModel, ContactUsModel,
+  Announcement, Advertisement, Bill, FlatModel, Service,

```

Comment:
Change Announcement to AnnouncementModel, Advertisement to AdvertisementModel, Bill to BillModel, Service to ServiceModel